web/template/payments.gohtml

```diff
@@ -56,6 +56,13 @@
                                     </a>
                                 </th>
                             </tr>
+                            <tr class="govuk-table__row">
+                                <th scope="row" class="govuk-table__header table-row__no-border">
```

Comment:
Probably best to add `colspan="2"` here so it spans across both columns, since it's full width.
```suggestion
                                <th scope="row" class="govuk-table__header table-row__no-border" colspan="2">
```